pkg/sdk/go/certs.go

```diff
@@ -20,33 +20,38 @@ const (
 
 // Cert represents certs data.
 type Cert struct {
-	ThingID    string    `json:"thing_id,omitempty"`
-	CertSerial string    `json:"cert_serial,omitempty"`
-	ClientKey  string    `json:"client_key,omitempty"`
-	ClientCert string    `json:"client_cert,omitempty"`
-	Expiration time.Time `json:"expiration,omitempty"`
+	SerialNumber string    `json:"serial_number,omitempty"`
+	Certificate  string    `json:"certificate,omitempty"`
+	Key          string    `json:"key,omitempty"`
+	Revoked      bool      `json:"revoked,omitempty"`
+	ExpiryDate   time.Time `json:"expiry_date,omitempty"`
+	EntityID     string    `json:"entity_id,omitempty"`
```

Comment:
We can keep ThingID here, just take care you serialize it properly (JSON tag should be `entity_id` and give a comment explaining the mapping.

Reply:
Update this in the rest of the codebase and API.